public/javascripts/display-paragraph.js

```diff
@@ -0,0 +1,16 @@
+// this function should display paragraph from database to appropriate story on the DOM
+
+exports.displayParagraph = (paragraph) => {
```

Comment:
The display paragraph method here is doing 2 things: 
- retrieve paragraph from database (data layer)
- prepare paragraph for display (view layer)

I’d recommend splitting these up into two different methods: one for paragraph retrieval that returns the paragraph, and one that prepares the paragraph for the view. I’d place the database related functions in a folder like JavaScripts/data/paragraph.js 

I’m not super familiar with the templating engine, pug, but you could leave the prepare paragraph method in this file for now

Reply:
Sounds good. I did end up doing this.